src/main/resources/site/processors/add-metadata.js

```diff
@@ -9,8 +9,14 @@ const view = resolve("add-metadata.html");
 
 exports.responseProcessor = function (req, res) {
     const site = libs.portal.getSite();
-    const content = libs.portal.getContent();
     const siteConfig = libs.common.getTheConfig(site);
+    const contentOrNull = libs.portal.getContent();
+
+    /** We don't want it 500 on controllers without content */
+    const content = contentOrNull ? contentOrNull : {
+        data: {},
+        x: {}
+    };
```

Comment:
How does this solve the issue? The rest of the functionality depends on content being not null, so shouldn't we just break or return empty response if it's null?